addons/bestja_frontend_fpbz/templates/snippets/get_knowledge.xml

```diff
@@ -17,30 +17,30 @@
                                 </div>
                                 <div class="col-md-10 centered">
                                     <div class="col-md-3 centered">
-                                        <div class="media_iframe_video" data-src="//www.youtube.com/embed/AuqxJgJTwaI?autoplay=0" style="">
+                                        <div class="media_iframe_video" data-src="http://www.youtube.com/embed/AuqxJgJTwaI?autoplay=0" style="">
```

Comment:
These `http` URLs to Youtube embed codes return 301 responses with redirects to `https` version, so we should put `https` directly here.